todo_app/data/trello_items.py

```diff
@@ -5,76 +5,65 @@
 load_dotenv()
 trello_key=(os.getenv('TRELLO_KEY'))
 trello_token=(os.getenv('TRELLO_TOKEN'))
-#Get Trello items
-def get_items():
-   
-  url = "https://api.trello.com/1/boards/62f2b96de0b99c00aff27ae6/lists?"
+trello_todo_list=(os.getenv('TRELLO_TODO_LIST'))
+trello_done_list=(os.getenv('TRELLO_DONE_LIST'))
+trello_board_url=(os.getenv('TRELLO_BOARD_URL'))
```

Comment:
This works for now, but might catch you out later when we look at testing. The reason is broadly as follows:
* When setting up tests, we sometimes want to override the environment variables
* But that code usually executes _after_ we've run our imports, and these statements here will get run when the file is first imported, so the variables will be set _before_ we override the values 😞 
* The test then updates the environment variables, but your functions below use the original values because they'd already been set

Thankfully it's easy enough to fix - we can just load the environment variables when we're actually executing the code, rather than in advance. So for example, we just load the variables inside the functions themselves:
```
  ...
  query = {
    'key': os.getenv('TRELLO_KEY'),
    'token': os.getenv('TRELLO_TOKEN'),
    'cards': 'open',
    'card_fields':'name'
    }
```
etc.